aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/IndicationProviderValueException.java

```diff
@@ -0,0 +1,12 @@
+package ru.fix.aggregating.profiler;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+public class IndicationProviderValueException extends RuntimeException {
```

Comment:
Maybe it would be better to give name: IndicationProviderRetrieveValueException

Reply:
ok